.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ workflows:
           name: docker-publish
           requires:
             - test
-          image: pennlabs/example-django
+          image: pennlabs/{{ project_name }}
```

Comment:
I don't think we'll always want the docker image name to be the same as the project name. For example with clubs, the backend image is called `clubs-backend`, but the project name is just `pennclubs`

Reply:
does it necessarily matter for new projects? this could start a convention of keeping image names the same as project names

Reply:
This is the template-django project, not the unified project. Unified project will have separated image names.

Reply:
We currently have CORS/CSRF configuration in this repo that only applies to unified projects. We should probably make a decision on how we want to split things up. I think it probably makes sense to have all (backend only and unified) django configuration in this repo, but then just have a note to delete certain things for backend only. The k8s probably should be specific to backend only, since we need to provide a new chart for a unified product anyway.

Reply:
yeah i agree that there is normally an associated frontend with a backend. don't know if that changes helm stuff or timelines @pawalt